index.html

```diff
@@ -210,10 +210,11 @@ <h2>Methods</h2>
     </p>
 
     <p class="advisement">
-      Developers are encouraged to utilize the [[DID-RUBRIC]] as early as poosible
-      in the development process to ensure that they are evaluating their DID method 
-      appropriately by the requirements as given by the problem they are attempting 
-      to solve with the introduction of a new DID method.
+      DID method developers are encouraged to review and apply the
+      [[DID-RUBRIC]] as early as possible in their development process to
+      ensure that they are evaluating their DID method appropriately by the
```

Comment:
```suggestion
      ensure that they are evaluating their DID method appropriately for the
```
There's always another tweak...

Reply:
always!